.travis.yml

```diff
@@ -67,6 +63,12 @@ before_script:
 script:
   - |
     if [[ $TRAVIS_EVENT_TYPE != "cron" ]]; then
+      if [[ $TOXENV == "amo-locales-and-signing" ]]; then
+         curl -sL -o ./gimme https://raw.githubusercontent.com/travis-ci/gimme/v1.2.0/gimme
+         chmod +x ./gimme
+         eval "$(./gimme 1.8.5)"
+         ./scripts/start-autograph.sh
+      fi
       RUNNING_IN_CI=True tox -v
     fi
   - |
```

Comment:
We should totally move this into a bash script to make it easier to maintain and test imho. Can very well be a follow-up issue though

Reply:
Moved to `start-autograph.sh` in 5d095ca